composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6,<7.4",
+        "php": "^5.6|^7.0",
```

Comment:
for no warnings requires @toooni PR https://github.com/ongr-io/ElasticsearchBundle/pull/906